### PR TITLE
feat: migrate to `unplugin`

### PR DESCRIPTION
**Pull request by elonehoo**
### Description

move to `unplugin`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,22 +92,24 @@
...
+    "tsx": "^3.12.7",
     "typescript": "^5.2.2",
-    "unbuild": "^2.0.0",
     "vite": "^4.4.9",
-    "vitest": "^0.34.3"
-  },
-  "peerDependencies": {
-    "vite": "^2.9.13 || ^3.0.0 || ^4.0.0"
-  },
-  "packageManager": "pnpm@8.7.0"
```
[the remark this answers is not in the document]
Comment (elonehoo) [reply package.json:72]: [b448297](https://github.com/yjl9903/vite-plugin-info/pull/313/commits/b448297c28207d4c4d6fabfbdffa33abe74dfdc5)

File: scripts/postbuild.ts
Comment (yjl9903) [inline scripts/postbuild.ts:1]: Why you replace tsup with unbuild, and add this manual fix? Is there any known issues?

Comment (elonehoo) [reply scripts/postbuild.ts:1]: 🤔, for this question, I think I can only say that I saw that [antfu](https://github.com/antfu) [unplugin template](https://github.com/unplugin/unplugin-starter) uses [tsup](https://github.com/egoist/tsup), but I have not tried using [unbuild](https://github.com/unjs/unbuild) yet.

**Comment by yjl9903** [comment]
Sorry for the late merge

Resolve #256

**Comment by elonehoo** [comment]
It’s okay, I’ve convinced my boss to use Vite

**Pull request merged**